ImageSegmentation/app/build.gradle

```diff
@@ -0,0 +1,35 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.2"
+
+    defaultConfig {
+        applicationId "com.example.imagesegmentation"
```

Comment:
For the first demo we used package org.pytorch.demo
Could it be also here, something like `org.pytorch.demo.imagesegmentation` ?